app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -128,6 +130,17 @@ class BrowserFragment : BaseBrowserFragment(), BackHandler {
         super.onStart()
         subscribeToTabCollections()
         getSessionById()?.register(toolbarSessionObserver, this, autoPause = true)
+        updateEngineBottomMargin()
+    }
+
+    private fun updateEngineBottomMargin() {
+        val browserEngine = swipeRefresh.layoutParams as CoordinatorLayout.LayoutParams
+
+        browserEngine.bottomMargin = if (requireContext().settings().shouldUseBottomToolbar) {
+            requireContext().dimen(R.dimen.browser_toolbar_height)
+        } else {
+            0
+        }
```

Comment:
Is this margin still needed?

Reply:
@pocmo Yes, if I don't do this, the GV will be covered by the bottom toolbar.